student.rb

```diff
@@ -0,0 +1,10 @@
+module Student
+  def initialize(classroom, age, parent_permission: true, name: 'Unknown')
+    super(name, age, parent_permission)
```

Comment:
- [ ] The order in which you give parameters to the super need to be the same as in the Person component. Right now in the Person component name is being placed where age is supposed. Kindly alter this piece of code in both the Student and Teacher classes